iis-common/src/main/java/eu/dnetlib/iis/common/model/conversion/ConfidenceAndTrustLevelConversionUtils.java

```diff
@@ -0,0 +1,63 @@
+package eu.dnetlib.iis.common.model.conversion;
+
+import eu.dnetlib.iis.common.InfoSpaceConstants;
+
+import java.util.Objects;
+
+/**
+ * Allows to convert confidence level to trust level and backwards.
+ */
+public class ConfidenceAndTrustLevelConversionUtils {
+    private static final float confidenceToTrustConversionFactor = InfoSpaceConstants.CONFIDENCE_TO_TRUST_LEVEL_FACTOR;
+    private static final float trustToConfidenceConversionFactor = 1 / InfoSpaceConstants.CONFIDENCE_TO_TRUST_LEVEL_FACTOR;
+
+    private ConfidenceAndTrustLevelConversionUtils() {
+    }
+
+    /**
+     * Converts confidence level to trust level using confidence to trust level scaling factor based on
+     * {@link InfoSpaceConstants#CONFIDENCE_TO_TRUST_LEVEL_FACTOR}. Provides null safety.
+     *
+     * @param confidenceLevel Confidence level to be converted.
+     * @return Float with corresponding trust level value.
+     */
+    public static Float confidenceLevelToTrustLevel(Float confidenceLevel) {
+        return Objects.nonNull(confidenceLevel) ? confidenceLevel * confidenceToTrustConversionFactor : null;
```

Comment:
I think a good rule of thumb in such cases, regardless the simplicity of the logic, is to avoid duplicating the code and relying on simple call to the `confidenceLevelToTrustLevel()` method accepting primitives. This would make future refactoring also much simpler.